maturity-level-two/README.md

```diff
@@ -14,6 +15,24 @@ You should:
 - Validate changes to your OpenAPI specs to avoid specification violations ([user guide](docs/validating-open-api-specs.md))
 - Unit test Open API validation to automatically detect breaking changes
 
+ ## Content negotiation
+With content negotiation a consumer specifies in which format he/she will communicate (send and receive data) with the server. In here the you, the API consumer, can specify what format you would like to receive from the server and send to to server:
+- `Content-Type` - Specify the format of the request
+- `Accept` - Specify the requested format of the response. Default format will be used when not  
+
+When you send a `Content-Type` the server doesn't understand, the server should return an HTTP 415: Unsupported Media Type. If the server cannot respond to your request in an `Accept` format, it will return an HTTP 406: Not Acceptable
+
+When adding Content-Negotiation to your project you should:
+* Think whether content negotiation is really necessary. For most of the cases you only need JSON and thus no content negotiation.
+* Remove input and output formatters when multi-format (JSON, XML, CSV, ...) is not necessary. 
+* Carefully evaluate whether you should use the [Produces] and [Consumes] attributes to further restrict the request and response formats for one specific acion or controller.
+    * [Produces] and [Consumes] are not meant to document the supported media types.  
+    * It is strongly advised to not use these attributes if you are supporting more than one media type (e.g. application/json and application/problem+json)
+    
+Notes: 
+* `Content-Type` is not needed for HTTP GET requests since a GET request has no request body
+* If you want to explicetly specify what content types are produced/consumed in your swagger file, we advise to use a custom attribute (to be checked whether something).                  
```

Comment:
The sentence is not completed. We should check whether the new version of swashbuckle introduce an attribute to specify produces and consumes without having a runtime impact